.github/workflows/NbodyGradient.yml

```
@@ -15,29 +15,29 @@ on:
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    name: Julia ${{ matrix.julia-version }} - ${{ matrix.os }} - ${{ matrix.julia-arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
-        version:
-          - '1.6'
-          - '1'
-        os:
-          - ubuntu-latest
-          - macOS-latest
-        arch:
-          - x64
+        julia-version: ['1', 'lts']
+        julia-arch: [x86, aarch64]
+        os: [ubuntu-latest, macOS-latest]
+        exclude:
+          - os: macOS-latest
+            julia-arch: x86
+          - os: ubuntu-latest
+            julia-arch: aarch64
```

Just remove the architecture, no?
```suggestion
        os: [ubuntu-latest, macOS-latest]
```

---

Ah, I suppose it would default to those, wouldn't it... Thanks!